src/components/PollBody/PollBody.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./PollBody.css";
+
+function PollBody({ pollData, setPollData, showCloseIcon }) {
+  function handleClosePollBody() {
+    setPollData({
+      question: "",
+      option1: "",
+      option2: "",
+      option3: "",
+      option4: "",
+      showPoll: false,
+    });
+  }
+
+  return (
+    <div className="poll-body-container">
+      <div className="d-flex justify-cont-between mb-1">
+        {pollData.question && (
+          <p className="pollbody-question">{pollData.question}</p>
+        )}
+        {showCloseIcon && (
+          <i
+            class="fa-solid fa-circle-xmark pollbody-close-icon"
+            onClick={() => handleClosePollBody()}
```

Comment:
no need to add callback just on click can do `onClick={handleClosePollBody}`

Reply:
Yupp